### DISCUSSION TITLE
Issue viewing data in Table

### DISCUSSION THREAD

**Discussion by ndlang** [Budibase/budibase#5380] [Help] [answered]
Is there a way tot formate the columns so more data is shown:
![image](https://user-images.githubusercontent.com/17033357/163067111-1f2a484f-494f-4443-bf57-02ee1b844e4c.png)

**Comment by melohagan** [comment, marked as answer]
Hi @ndlang 

Yes you can! https://docs.budibase.com/docs/table#advanced-column-customization

**Reply by ndlang** [reply]
Hey @melohagan,

Thank you for the reply, but I  must be missing something I am not seeing the advanced column settings:
![image](https://user-images.githubusercontent.com/17033357/163882774-d14b4450-1a3b-4333-b524-4a4bc5901feb.png)
Can you point me in the right direction?

**Reply by ndlang** [reply]
I have tired in Safari and Firefox and still am not seeing the advanced options.

**Comment by gerhardmpl** [comment]
@ndlang is your Budibase installation up-to-date / which version are you running? From the settings panel, it looks as if your are some versions behind. Not sure (beeing a Budibase noob), but might be the reason why the configure columns button is missing.

**Reply by ndlang** [reply]
Current version
0.9.186

**Reply by ndlang** [reply]
I am a BB noob as well.

**Comment by gerhardmpl** [comment]
The Advanced Column Customization feature was introduced with the February release (see #4949) so I think that you need to update your Budibase installation. Reading #4249 on pre V1.x updates, I would be careful and export important apps before doing a `budi hosting --update`. I never had problems updating, but you never know. Current BB version is 1.0.110.

**Reply by ndlang** [reply]
Hey thanks for the info...updating now. I appreciate the quick response. Ill let you know if it fixes the issue.

**Comment by ndlang** [comment]
So, I am still not seeing the advanced options:
![image](https://user-images.githubusercontent.com/17033357/163887025-1cd671ec-e334-4295-b3d8-aceafbb01c4d.png)

Current version
1.0.110

**Reply by gerhardmpl** [reply]
@ndlang I see that the update app icon seems to be blue which indicates that most likely the app itself also needs to be updated (by clicking on that icon). If full up-to-date, the icon looks like this:
![grafik](https://user-images.githubusercontent.com/40722408/163888296-b6c42a54-b51b-4baf-af0f-1bc2c7d43a9c.png)

**Reply by ndlang** [reply]
Good catch.

**Reply by melohagan** [reply]
@gerhardmpl Thanks for helping out! :)

@ndlang Were you able to format the column the way you wanted? Did the docs help?
